### PR TITLE
Add docker-compose deployments for all supported blockchains until no…

### LINKED ISSUE #12: Create docker-compose for all blockchains currently available for pocket-core
Create folder with all blockchains supported by pocket-core with their own docker-compose.yml for minimal deployment

### PR DESCRIPTION
**Pull request by aqt01**
- Add docker-compose for all blockchains supported by pocket-core
- Add minimal docks on how to run them
- Adds nginx as additional container for using proxy
- Not recommended for running in production, needs some improvements, but its a great tool for testing all of them

Closes #12

### CODE REVIEW

**Review by luyzdeleon — CHANGES_REQUESTED** [review]
The directory structure for this is wrong, we need to have the deployment artifact platform as the root folder:

dappnode
docker
docker-compose
etc...

File: docker-compose/README.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+# Pocket-core docker-compose
```
Comment (luyzdeleon) [inline docker-compose/README.md:1]: Can this README follow the same format as here: https://github.com/pokt-network/pocket-core-deployments/blob/mvp-master/dappnode/README.md

And just link to the developer portal here: https://docs.pokt.network/docs/docker-service-node-setup

File: docker-compose/aion/aion-mastery/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,2 @@
+FROM aionnetwork/aionr:Latest
```
Comment (luyzdeleon) [inline docker-compose/aion/aion-mastery/Dockerfile]: Is there a typo on `Latest`, should it be `latest`?

File: docker-compose/all/docker-compose.yml [hunk 3]
```diff
@@ -0,0 +1,174 @@
+version: '3'
```
Comment (luyzdeleon) [inline docker-compose/all/docker-compose.yml:1]: Can we add a comment at the top of the file indicating which networks are included in `all`?

File: docker-compose/nginx/Dockerfile [hunk 4]
```diff
@@ -0,0 +1,155 @@
+FROM alpine:3.9
```
Comment (luyzdeleon) [inline docker-compose/nginx/Dockerfile:1]: Can we explain either in the README or in this same file why we need nginx? Whether is required or optional?

**Review by luyzdeleon — CHANGES_REQUESTED** [review]
Can we add to the top of every docker-compose file what is being installed and how to configure it?

**Review by luyzdeleon — DISMISSED** [review]

File: docker-compose/lightchain/Dockerfile [hunk 5]
```diff
@@ -0,0 +1,22 @@
+FROM golang:1.10-stretch
+
+MAINTAINER Gabriel Garrido
```
Comment (edjroz) [inline docker-compose/lightchain/Dockerfile]: The `MANTAINER` keyword for docker files has been marked as [deprecated](https://docs.docker.com/engine/reference/builder/) and should likely be replaced by `LABEL`

**Review by nelcolon — DISMISSED** [review]

**Review by edjroz — DISMISSED** [review]

**Review by nelcolon — COMMENTED** [review]
Should be fine.

**Review by nelcolon — APPROVED** [review]

**Pull request merged**